pkg/kubehound/storage/graphdb/janusgraph_provider.go

```diff
@@ -67,50 +67,63 @@ func (jgp *JanusGraphProvider) Prepare(ctx context.Context) error {
 		return nil
 	}
 
-	g := gremlin.Traversal_().WithRemote(jgp.drc)
-	tx := g.Tx()
-	defer tx.Close()
-
-	for {
-		// Begin a new transaction.
-		gtx, err := tx.Begin()
-		if err != nil {
-			return err
-		}
-
-		// Retrieve the number of vertices in the graph.
-		page, err := gtx.V().Count().Next()
-		if err != nil {
-			return err
-		}
-
-		// Decode the number of vertices from the page.
-		count, err := page.GetInt()
-		if err != nil {
-			return err
-		}
-
-		// If there are no more vertices to delete, break the loop.
-		if count == 0 {
-			break
-		}
-
-		// Delete the vertices in the graph.
-		err = <-gtx.V().Limit(deleteBatchSize).Drop().Iterate()
-		if err != nil {
-			return err
-		}
-
-		// Commit the transaction.
-		if err := tx.Commit(); err != nil {
-			return err
-		}
+	// These vertex types are defined in the schema.
```

Comment:
Small nit:
Is there a way to get this list from somewhere?
I worry about adding a vertex type in the future and forgetting this (which could break some queries in subtle way with duplicated entries for example?)